bundles/org.openhab.binding.neohub/src/test/java/org/openhab/binding/neohub/test/NeoHubTestData.java

```diff
@@ -17,197 +17,372 @@
-            + "{\"AWAY\":false,\"COOLING\":false,\"COOLING_ENABLED\":false,\"COOLING_TEMPERATURE_IN_WHOLE_DEGREES\":0,\"COOL_INP\":true,\"COUNT_DOWN_TIME\":\"0:00\",\"CRADLE_PAIRED_TO_REMOTE_SENSOR\":false,\"CRADLE_PAIRED_TO_STAT\":false,\"CURRENT_FLOOR_TEMPERATURE\":0,\"CURRENT_SET_TEMPERATURE\":\"0.0\",\"CURRENT_TEMPERATURE\":\"0.0\",\"DEMAND\":false,\"DEVICE_TYPE\":5,\"ENABLE_BOILER\":false,\"ENABLE_COOLING\":false,\"ENABLE_PUMP\":false,\"ENABLE_VALVE\":false,\"ENABLE_ZONE\":false,\"FAILSAFE_STATE\":false,\"FAIL_SAFE_ENABLED\":false,\"FLOOR_LIMIT\":false,\"FULL/PARTIAL_LOCK_AVAILABLE\":false,\"HEAT/COOL_MODE\":false,\"HEATING\":false,\"HOLD_TEMPERATURE\":0,\"HOLD_TIME\":\"0:00\",\"HOLIDAY\":false,\"HOLIDAY_DAYS\":0,\"HUMIDITY\":0,\"LOCK\":false,\"LOCK_PIN_NUMBER\":\"0000\",\"LOW_BATTERY\":true,\"MAX_TEMPERATURE\":\"0.0\",\"MIN_TEMPERATURE\":\"0.0\",\"MODULATION_LEVEL\":0,\"NEXT_ON_TIME\":\"0 days 00:00\",\"OFFLINE\":0,\"OUPUT_DELAY\":false,\"OUTPUT_DELAY\":0,\"PREHEAT\":false,\"PREHEAT_TIME\":\"0:00\",\"PROGRAM_MODE\":\"NONPROGRAMMABLE\",\"PUMP_DELAY\":false,\"RADIATORS_OR_UNDERFLOOR\":false,\"SENSOR_SELECTION\":\"BUILT_IN_AIR_SENSOR\",\"SET_COUNTDOWN_TIME\":0,\"STANDBY\":false,\"STAT_MODE\":{\"4_HEAT_LEVELS\":true,\"MANUAL_OFF\":true,\"TIMECLOCK\":true},\"TEMPERATURE_FORMAT\":false,\"TEMP_HOLD\":false,\"TIMECLOCK_MODE\":false,\"TIMER\":false,\"TIME_CLOCK_OVERIDE_BIT\":false,\"ULTRA_VERSION\":0,\"VERSION_NUMBER\":0,\"WRITE_COUNT\":0,\"ZONE_1PAIRED_TO_MULTILINK\":false,\"ZONE_1_OR_2\":false,\"ZONE_2_PAIRED_TO_MULTILINK\":false,\"device\":\"Nursery Window\"}]}";
+    /*
+     * Load the test JSON payload string from a file
+     */
+    private String load(String fileName) {
+        try (FileReader file = new FileReader(String.format("src/test/resources/%s.json", fileName));
+                BufferedReader reader = new BufferedReader(file)) {
+            StringBuilder builder = new StringBuilder();
+            String line;
+            while ((line = reader.readLine()) != null) {
+                builder.append(line).append("\n");
+            }
+            return builder.toString();
+        } catch (IOException e) {
+            fail(e.getMessage());
+        }
+        return "";
+    }
 
+    /*
+     * The main unit test method
+     */
     @Test
     public void test() {
-        // info response (old JSON format)
-        NeoHubInfoResponse infoResponse = NeoHubInfoResponse.createInfoResponse(NEOHUB_JSON_TEST_STRING);
+        testInfoJsonOld();
+        testInfoJsonNew();
+        testReadDcbJson();
+        testInfoJsonWithSensors();
+        testGetSystemJson();
+        testGetLiveDataJson();
+        testGetEngineersJson();
+        testCommunications();
```

Comment:
Rather than a "main" unit test method I would prefer if each of these methods were their own unit test. 
I guess this would be ok if each of these tests rely on the state left by a prior state. If that is not the case please add a `@Test` annotation to each of these test methods.

Reply:
Ok. { I will do this in [siemensrds] as well.. }